### ISSUE TITLE
Feature Request - dome9_continuous_compliance_policy Organizational Unit support

### ISSUE THREAD

**Issue by cwiegleb** [dome9/terraform-provider-dome9#83]
Hi folks, 

we as a team really enjoy the automation with Terraform and the fact that you not only offer an API. But unfortunately we recognized that not all of your resources support the target type `OrganizationalUnit`. 

### Is your proposal related to a problem?

At the moment we setup our organizational structure in Dome9 to avoid (AWS) account berry picking for compliance checks. But the resource `dome9_continuous_compliance_policy` doesn't support the target type 'OrganizationalUnit` like your [API V2 ](https://api-v2-docs.dome9.com/?shell#Dome9-API-ContinuousCompliancePolicyV2) it does. 

### Describe the solution you'd like

It would be great to have something similar to your API V2: 

```
resource "dome9_continuous_compliance_policy" "test_policy" {
  target_id = "CLOUD ACCOUNT ID"
  bundle_id = 00000
  target_type = "OrganizationalUnit"
  notification_ids = ["NOTIFICATION IDS"]
}
```

Cheers and thanks, 
Christian

**Comment by erezweinstein5-zz** [comment]
@cwiegleb thanks for using our provider and sorry for the late response.
The continuous compliance policy V2 API was released after we published this resource (which works against V1 API endpoint). This issue will remain open and it will be added to the roadmap with higher priority.

Greetings,
Erez

**Comment by erez-dome9** [comment]
Hi @cwiegleb . resolved in #89

**Comment by erezweinstein5-zz** [comment]
@cwiegleb this feature is currently under development. Stay tuned :)

**Comment by erezweinstein5-zz** [comment]
resolved in #91
